### DISCUSSION TITLE
Teambuilding event in Hubs

### DISCUSSION THREAD

**Discussion by Kanishk-Malick** [mozilla/hubs#2861] [General Questions & Discussion] [answered]
Can we do a team building event in Hubs ? If yes, what is the level of interaction & immersiveness can we achieve ? Has anyone tried doing it ?

**Comment by aziliak** [comment]
Yes.  This is one of the main things Hubs is directed towards.  There are weekly meetings in Hubs where we all get together to discuss issues with Hubs and have show and tells where we take the whole group to different environments.

I personally think Hubs is way better than Zoom meetings.  You can have side discussions with people, share your screen, pull up pdfs/images, share your webcam.  I only ever use my computer to attend the meetings because I often record them in OBS and make notes of things during the meetups.

If you are looking to organize an event for your team and possibly create a plan for continued meetings, get in contact with me and I can see if we can develop something for you.  Feel free to send me a message at info@lucidimmersions.com.

**Comment by bemyevent** [comment]
We have done an escape room with enigma on hubs,  you can use it for team building

**Comment by Kanishk-Malick** [comment]
Can you share the link please?

**Reply by Kanishk-Malick** [reply]
Also, how is this being done on a different URL than hubs.mozilla.com ? Sorry, if I am asking naïve questions, I'm new here.

**Reply by emclaren** [reply]
Hi! at the moment you can use Hubs in two ways 

1) on the Mozilla hosted server: hubs.mozilla.com 
2) on your own personal server using Hubs Cloud. When you host it on your own server, you can use your own custom domains. There is more information on the differences outlined in this discussion: https://github.com/mozilla/hubs/discussions/2725

**Comment by bemyevent** [comment]
![Screenshot_20200816-150535_Video Player](https://user-images.githubusercontent.com/58043917/90335022-1401a600-dfd2-11ea-9709-78d2b18eb2fa.jpg)
![Screenshot_20200816-150526_Video Player](https://user-images.githubusercontent.com/58043917/90335023-149a3c80-dfd2-11ea-80e8-d57744573072.jpg)

[*Image preview hidden because of graphic content, click through to view](https://user-images.githubusercontent.com/58043917/90335024-1532d300-dfd2-11ea-834a-13dfba1168b2.jpg)

![Screenshot_20200816-150613_Video Player](https://user-images.githubusercontent.com/58043917/90335025-1532d300-dfd2-11ea-94c4-7d228d2e9222.jpg)
![Screenshot_20200816-150552_Video Player](https://user-images.githubusercontent.com/58043917/90335026-15cb6980-dfd2-11ea-810d-358f21bd3e5a.jpg)
![Screenshot_20200816-150605_Video Player](https://user-images.githubusercontent.com/58043917/90335027-15cb6980-dfd2-11ea-8b16-51a801e302d6.jpg)
Just can share Some screenshot for the moment .

* Note: @emclaren edited this comment to remove the preview of graphic content as per Github's [Community Guidelines](https://docs.github.com/en/github/site-policy/github-community-guidelines)

**Comment by bemyevent** [comment]
The user can discover a large house with 2 stairs ansd résolve enigma .

**Comment by misslivirose** [comment, marked as answer]
It's really up to the communities that are using Hubs how they want to structure their time using the platform, but generally, we hear that Hubs is great for team-building! We've used Hubs to host casual events on our team where we just hang out and chat, play Jackbox games together, watch videos, go on tours of scenes and experiences that other community members have made, or share things we're working on in a more free-form way than on video chat. 

Depending on how invested in using Hubs you want to go, you can also experiment with adding your own interactive elements, and adding more interactive content to Hubs is something that is on our roadmap. The escape room on this thread is a good example of a way that a custom experience can be created depending on the desired experience. 

As far as immersion goes, Hubs works for people using VR and non-VR, and this really depends on the individual, but we get feedback that even in desktop mode, using Hubs has more of a sense of shared presence with others since you have a virtual space together. VR is generally considered more immersive than viewing in "flat" mode, but both offer a different feature set compared to video calling.

**Reply by Kanishk-Malick** [reply]
"experiment with adding your own interactive elements" Please explain more on this, interactive elements, such as ?
Really appreciate your response.

**Comment by Kanishk-Malick** [comment]
Hi,
I went thorugh the website https://stealink-room.com/
Quite interesting rooms there. Can you tell me more about it ?

Warm Regards,
Kanishk Malick

**Comment by Kanishk-Malick** [comment]
Can I visit the room ?

Warm Regards,
Kanishk Malick
